src/@carbon-info/views/IntroPage/IntroPage.tsx

```diff
@@ -167,7 +167,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     color: theme.palette.text.hint,
   },
   altCarbonContainer: {
-    top: "70vh",
+    top: "75vh",
```

Comment:
Think it's good not to keep it as `70vh` as `75vh` would make the link go out of frame for some dimensions.
<img width="1440" alt="Screenshot 2023-08-16 at 7 29 26 PM" src="https://github.com/Switcheo/carbon-website/assets/65583522/b95d176b-28ab-463f-b9fa-ab1031465478">

Reply:
Okay!